### ISSUE TITLE
doom-emacs configuration

### ISSUE THREAD

**Issue by OrionRandD** [rksm/org-ai#50]
I have added:

(package! org-ai) to "packages.el"

and the configuration bellow in my 
"config.el" file.

(use-package! org-ai
  :ensure t
  :commands (org-ai-mode
             org-ai-global-mode)
  :init
  (add-hook 'org-mode-hook #'org-ai-mode) ; enable org-ai in org-mode
  (org-ai-global-mode) ; installs global keybindings on C-c M-a
  :config
  (setq org-ai-default-chat-model "gpt-3.5-turbo") ; if you are on the free gpt-3.5-turbo:
  ;; (setq org-ai-default-chat-model "gpt-4") ; if you are on the gpt-4 beta:
  (org-ai-install-yasnippets)) ; if you are using yasnippet and want `ai` snippets

in my doom-config files.

org-ai has been installed with no errors, but it does nothing inside an org-ai block     :(

I have tried these two styles, putting my point in the middle of the block and hitting Enter key, but no answers from the chatbot:
Could you include a  doom-emacs working sample configuration example in instructions? If possible?
Thx a lot for the wonderful package...

hash+begin_ai
[SYS]: You are a helpful assistant.

[ME]: 
hash+end_ai

hash+BEGIN_SRC ai :results replace
[SYS]: You are a helpful assistant.

[ME]: 
hash+END_SRC

I've written "hash" for not to mess up the message here, since it is a markdown tag...

**Comment by activeli** [comment]
```lisp
(package! org-ai
  :recipe (:host github
           :repo "rksm/org-ai"
           :files ("*.el" "README.md" "snippets")))

```

**Comment by OrionRandD** [comment]
When I am inside a org-ai block and hit ENTER in doom-emacs, it gives me this error:

begin_ai
[SYS]: You are a helpful assistant.

[ME]: 
end_ai

==> ERROR ==> org-ai-complete-block: Symbol’s function definition is void: string-equal-ignore-case 

and no answer at all...

note: I have omitted the hash-plus in the beggining of the block

;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;

my config is as follow:

in config.el:

;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
;; org-ai 
;; chatgpt
;; dall-e
;; whisper
;; https://github.com/rksm/org-ai

;; aqui
;; fixIT

 (use-package! org-ai
   :ensure t
   :defer t
   :commands (org-ai-mode
              org-ai-global-mode);;
   :init
   (add-hook 'org-mode-hook #'org-ai-mode) ; enable org-ai in org-mode
   (org-ai-global-mode)                  ; installs global keybindings on C-c M-a
   :config
   (setq org-ai-default-chat-model "gpt-3.5-turbo") ; if you are on the free gpt-3.5-turbo:
   ;; (setq org-ai-default-chat-model "gpt-4") ; if you are on the gpt-4 beta:
   (org-ai-install-yasnippets)) ; if you are using yasnippet and want `ai` snippets

;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
;; whisper
;; prefer using "espeak"

;; https://github.com/natrys/whisper.el

;; (add-to-list 'load-path "~/.emacs.d/local-repo/whisper");;

;; (use-package greader
;;  :ensure t);;

;; (require 'whisper);;
;; (require 'org-ai-talk);;

;; ;; speech settings, optional
;; ;; (setq org-ai-talk-say-words-per-minute 200);;
;; (setq org-ai-talk-say-voice "Karen");;
;; (setq whisper-install-directory "~/.emacs.d/.cache"
;;         whisper-model "base"
;;         whisper-language "en"
;;         whisper-translate nil);;

;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;

in packages.el:

(package! org-ai
  :recipe (:host github
           :repo "rksm/org-ai"
           :files ("*.el" "README.md" "snippets")))

;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;

**Comment by OrionRandD** [comment]
> ```lisp
> (package! org-ai
>   :recipe (:host github
>            :repo "rksm/org-ai"
>            :files ("*.el" "README.md" "snippets")))
> ```

Thx. This worked for installing org-ai in doom-emacs. 
But, I am still having issue upon using the org-ai orgmode-block...

**Comment by rksm** [comment]
this seems like a duplicate of #53. Fixed in 0.3.6.

**Comment by rksm** [comment]
Should be fixed now, please reopen if not.
